examples/BUILD

```diff
@@ -23,6 +23,7 @@ filegroup(
             "opentelemetry/otel-collector-config.yaml",
             "**/*docker-compose*.yaml",
             # Contrib extensions tested over in contrib.
+            "dlb/*.yaml",
```

Comment:
please move this up a line and add a comment as to why the config cant be tested - its _not_ tested in contrib